README.md

```diff
@@ -16,6 +16,7 @@
 
 # Useful Documents
 
+- [Code of Conduct](./CODE_OF_CONDUCT.md)
```

Comment:
Can you alphabetise the links while youre at it?

Reply:
Oh yea, good idea.